azerothcore/azerothcore-wotlk
# fix(Core/player): do not show zone at first login

Do not show location at first login.

## CHANGES PROPOSED:
- if it is the first login send `0` in the world server packet as `zone` instead of the value of the zone

## ISSUES ADDRESSED:
- Closes #3354 

## TESTS PERFORMED:
- created new character and checked if the zone is showed or not at first login

## HOW TO TEST THE CHANGES:
- create new character
- check if the zone is showed at first login

## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
